pixel/theme.json

```diff
@@ -269,6 +269,13 @@
                 "typography": {
                     "fontSize": "var(--wp--preset--font-size--large)",
                     "lineHeight": "1.125"
+                },
+                "elements": {
+                    "link": {
+                        "typography": {
+                            "textDecoration": "none"
```

Comment:
In terms of link styles, the post title seems like more of an exception than the rule. Setting text decoration to none for all links seems heavy handed, since they are underlined by default in post content. What do you think?

Reply:
This setting is under `"core/post-title"`, so it shouldn't spill onto other links.

Reply:
Ah, my bad!